examples/ktor/src/main/resources/application.conf

```diff
@@ -1,7 +1,6 @@
 ktor {
   deployment {
     port = 8080
-    port = ${?PORT}
```

Comment:
Looks like standard configuration to support environment variables, what was the reason for removal?

Reply:
It does not seem to be necessary for an example on how to use the ktor integration, but I can put it back in if you prefer that.

Reply:
Fine for me, I was just wondering 😉